test/objectwrap.js

```diff
@@ -256,9 +256,19 @@ const test = (binding) => {
     testFinalize(clazz);
   };
 
+  const testConstructorExceptions = () => {
+    const TestConstructorExceptions = binding.objectwrap.TestConstructorExceptions;
+    console.log("Runnig test testConstructorExceptions");
+    assert.throws(() => { new TestConstructorExceptions(); });
+    global.gc();
+    console.log("Test testConstructorExceptions complete");
+  }
+
   // `Test` is needed for accessing exposed symbols
   testObj(new Test(), Test);
   testClass(Test);
+
+  testConstructorExceptions();
 }
 
 test(require(`./build/${buildType}/binding.node`));
```

Comment:
```suggestion
test(require(`./build/${buildType}/binding.node`));
test(require(`./build/${buildType}/binding_noexcept.node`));
```

Please test both scenarios!